news/2022-05-19-osu-talk-event-overcoming-obstacles.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "osu! Talk Event: Overcoming Obstacles"
+date: 2022-05-19 08:00:00 +0000
+---
+
+Mark your calendars, set your alarms, and join us for the first ever osu! talk event, where you can make your voice heard.
+
+![](/wiki/shared/news/2022-05-19-osu-talk-event-overcoming-obstacles/ote-newspost-banner.png)
+
+Unlike many events that involve guest speakers, this one is open for anyone to speak — all you need is a story to share. Each event centers around a certain theme. For this iteration, all talks will center around the theme of **overcoming obstacles**.
+
+Here's a rundown of how the event works:
+
+- **The event will last 3 days total**, from May 20 - 22, 2022.
+- **The speaker will use a Discord stage channel** in the [official osu!dev Discord server](https://discord.gg/ppy) to give their talk.
+- **Talks will be less than 18 minutes long.** This is the maximum allowed length, not a goal to aim for — even a 3-minute talk can have massive impact.
+- The stories shared do not necessarily have to be about osu!, but because it is an osu!-related event, the speakers are recommended to try connecting their story or the lesson behind their story to osu! in some way.
```

Comment:
```suggestion
- The stories shared do not necessarily have to be about osu!, but because this is an osu!-related event, speakers are recommended to try connecting their story or the lesson behind it to osu! in some way.
```